projects/ruamel-yaml/README.md

```diff
@@ -28,15 +28,40 @@ Change directory to `/fuzz`:
 
 ### yaml
```

Comment:
Here could just enumerate commands for every step without concrete fuzztarget, as its differs only by config name